examples/internal-load-balancer/files/main.csr

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE REQUEST-----
+MIIDSDCCAjACAQAwgZgxCzAJBgNVBAYTAlVTMQswCQYDVQQIDAJDQTEWMBQGA1UE
+BwwNU2FuIEZyYW5jaXNjbzEMMAoGA1UECgwDVEZFMREwDwYDVQQLDAhFeGFtcGxl
+czEfMB0GA1UEAwwWSW50ZXJuYWwgTG9hZCBCYWxhbmNlcjEiMCAGCSqGSIb3DQEJ
+ARYTaGVsbG9AaGFzaGljb3JwLmNvbTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCC
+AQoCggEBALH5GhrsMtyqssvUCaQfvYcS/Ig3n4XbrnkgQOG1dRqdDZ4GpnX+Mc8G
+WTc2sVSv/B0QMDbFdbKPwWua1HE3KUhUyP4stuwOhWJfYWEbqZiMLEauK4u1o+Lt
+GoeFxkG2k6hotejoXi3pGzeIPExyhtVp4ED585bBq9tF9lLakLc6KCFdy1TBiG0v
+6mW3kbqxXRFR2K0O33LukFc429sseSPSB0cC0ZvNF2y5nkPGrAoXJh84a0OmLtgj
+tiHQO3sqs6qWUl+l2C+UcFJsxd4LLBQjC+KKuE9zqKCXMnLxBbH8zNvXhLJTJRI3
+FEi8LFtd7OZoY8gz+7evO3KpG1PUXGECAwEAAaBqMGgGCSqGSIb3DQEJDjFbMFkw
+CQYDVR0TBAIwADALBgNVHQ8EBAMCBeAwPwYDVR0RBDgwNoIQdGZlaWxiLXByaW1h
+cnktMIIQdGZlaWxiLXByaW1hcnktMYIQdGZlaWxiLXByaW1hcnktMjANBgkqhkiG
+9w0BAQsFAAOCAQEAeitWfFrdVUeBvF92VHgoQaQr4BPaBdqIcr6wKnjfMCfs0doh
+zY0ke+k6/kBiosR1dzsQeq1YwcGdkvacwy+wsC8fBaTkBzpN8W6oKhI4K9OyjMYp
+Ag+UtN7ApusIXf24SAkz0m7gqvGvDjOdrvb66nlIuU+CL4S9O3JSeGKcsyGy6pDx
+e+GUtrRmcndtn23u17MS9HjNARhNvomlRzGsSkmDWlluJQuzGLJaCMjPQTI/9lkB
+ODsHHSqSNWfAT0BUGu9Wy5DUxhQKMxpGYyGzDSD9p69bUIDQ7/cmL1lPN4z0uKOb
+nnpOVjdUtFWajbAkISPAd6wl96K+OxkpqYiz1w==
+-----END CERTIFICATE REQUEST-----
```

Comment:
For this CSR, is there an expiry on the associated priv key and how will this impact future upgrades?

Reply:
Fixed by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/73/commits/fba16bc1d0358f0c82a9598e178e962d244104ec.